IDEAS/Buildings/Components/Zone.mo

```diff
@@ -6,46 +6,92 @@ model Zone "thermal building zone"
   extends IDEAS.Fluid.Interfaces.LumpedVolumeDeclarations(
     energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
     redeclare replaceable package Medium = IDEAS.Media.Air,
-    final mSenFac = corrCV);
+    final mSenFac = 1);
```

Comment:
I don't think so. Since mSenFac is defined in IDEAS.Fluid.Interfaces.LumpedVolumeDeclarations, which we are extending in that line of code, that is not even possible? You could set mSenFac=airModel.mSenFac, but airModel.mSenFac is only defined when it contains a mixing volume so actually that's not possible either.
Note that the value of mSenFac does not matter since it's not used. Actually it may be better to remove the extension of IDEAS.Fluid.Interfaces.LumpedVolumeDeclarations and to create a custom definition of the Medium instead..?